codegen/protocol-test-codegen-local/model/eventstream.smithy

```diff
@@ -1,4 +1,4 @@
-namespace aws.protocoltests.restjson
+namespace aws.protocoltests.eventstream
```

Comment:
This is the non-RPC EventStream model.

It is pre-existing, and was added when event streaming was first added to the SDK.